### ISSUE TITLE
In table HS_BULK_DATA two fields are truncated and a few more have minor data quality issues

### ISSUE THREAD

**Issue by sfc-gh-space** [starschema/COVID-19-data#148]
In table HS_BULK_DATA two fields are truncated and a few more have minor data quality issues:

- Name was truncated at 100 characters on import, appears to be more characters from source
- Operator was truncated at 100 characters on import, appears to be more characters from source
- BEDS should be numeric and could be converted now to NUMBER(38,0)
- STAFF_MEDICAL and STAFF_NURSING should really be numeric; however, they can't be converted because 7 records have the text ‘unknown’ instead of a valid number
- HEALTHCARE_PROVIDER_TYPE: all records are in English but 55 records are in German:
Rehaklinik für Orthopädie und rheumatische Erkrankungen -> Rehabilitation clinic for orthopedics and rheumatic diseases
- HEALTHCARE_PROVIDER_TYPE: One record has a value of * — is there a meaning for that? Perhaps should be blanked out

**Comment by tfoldi** [comment]
Regarding `name` and `operator`, we should check why we have only 100 cols

`Beds`, `Staff_Medical` and `Staff_Nursing` were fixed in 3d059f1 

`HEALTHCARE_PROVIDER_TYPE` - see we have a lot of unhandled values (still). @chrisvoncsefalvay - could you have a look at it?

**Comment by chrisvoncsefalvay** [comment]
Expected behaviour for this is for the data set to be filtered for things that can either be normalised or are not specified. The notebook discusses this as follows:

```md
We are also excluding
- indirect medical specialities (e.g. diagnostics),
- chronic care (e.g. dialysis),
- mental health provision incl. drug rehabilitation,
- labour and delivery,
- specialities that do not play a role in COVID-19 response (dentistry, optometry, physiotherapy &c.), and
- alternative 'medicine' and paramedicine.

Finally, we map various local names and different spellings into the following categories:
- primary_care (GPs, PCPs, paediatrics),
- secondary_care,
- hospitals incl. polyclinics,
- emergency_and_urgent_care,
- health_centers, incl. multispeciality care.
```

I have now implemented a filter in #155 that complies with these filtering criteria.
